docs/topic-guides/blendable-web-apps/uniform-design-system.md

```diff
@@ -0,0 +1,90 @@
+
+Unlike many design systems, Uniform is open-ended and themeable. It can be configured, extended or replaced in parts to achieve anything that is possible with HTML, CSS, and JavaScript.
+
+## Mental model of client-side rendering by the Starcounter host
+
+Starcounter host follows a layered design that separates the responsibilities of the app author, the design system and the solution owner. Uniform fits in this design as the middle layer. 
+
+The full design consists of:
+
+1. Native HTML5 and Shadow DOM
+2. Uniform design system
+3. Theme configuration through CSS custom properties
+4. Extend or replace the theme with HTML, CSS, and JavaScript
+
+### Native HTML5 and Shadow DOM
+
+As an app author, you provide the views for your apps with a clear separation of the content and the presentation.
+
+In the content part, you provide the UI components that should be rendered by the Starcounter host as unstyled native HTML5 elements. 
+
+To prevent the Starcounter host from controlling the rendering of a UI component, you can use shadow DOM, preferably in a custom element. The shadow root creates a boundary that blocks the Starcounter host's attempt to apply the look and feel on your components automatically.
+
+### Uniform design system
+
+Uniform design system gives a common look and feel to all apps. It consists of CSS custom properties, CSS reset and a library of patterns and components.
+
+The CSS custom properties are the basic set of primitives. They define the fonts, sizes, colors, margins and other values used throughout the reset, patterns and components.
+
+The CSS reset, nicknamed [Underwear.css](https://starcounter.github.io/underwear.css/), overwrites the default user agent stylesheet. It provides the look of the native HTML5 elements such as the headings (H1-H6), links, inputs, buttons, and tables. It is implicitly loaded by the Starcounter app shell, so there is no need to load it explicitly in the views. 
+
+The pattern library, [Uniform.css](https://starcounter.github.io/uniform.css/), is a set of CSS classes that implement visual hierarchy and give specific roles to native elements. It provides sections, cards, titles, column layouts, alerts,  buttons, etc.
+
+The component library, [Uniform components](https://starcounter.github.io/uniform.css/components/), is a set of custom elements that implement interactive UI controls. For the beginning, we provide a form item, a date picker, a data table and a pagination control. 
+
+To use the patterns and the components, you must import them explicitly. They are placed in the shadow root of the view container to wrap the distributed content elements. That allows to serve real progressive enhancement, and what is crucial for a solution of blended apps: keep app content's markup semantic and unopinionated, giving solution owner a way to apply own opinion without changing the app source code.
```

Comment:
I'd say "They are meant to be used" instead of "They are placed" because they aren't, you as an author (app dev or solution owner) need to place it there.